lib/liquid/extensions.rb

```diff
@@ -7,6 +7,12 @@ def to_liquid
   end
 end
 
+class Symbol # :nodoc:
+  def to_liquid
+    self
```

Comment:
Do we want to have to deal with the difference between symbols and strings in filters and tags?  Or should we use `to_s` here so that symbols get converted into strings and we don't have to worry about another possible type?